src/Assets/view/administrator/src/Table/table.jext

```diff
@@ -103,7 +103,7 @@ class {{singular_capitalize}}Table extends Table
 		$table = Table::getInstance('{{singular_capitalize}}Table', __NAMESPACE__ . '\\', array('dbo' => $this->getDbo()));
 		$table->load(array('alias' => $this->alias));
 
-		if ($table->load(array('alias' => $this->alias)) && ($table->id !== $this->id || $this->id === 0))
+		if ($table->load(array('alias' => $this->alias)) && ($table->id != $this->id || $this->id === 0))
```

Comment:
Same here, why `!=` instead of `!==`

Reply:
this is due to the fact that if you try to change a title you get an "alias already exists" error message

Reply:
I believe the `$table->id` and the `$this->id` would be the same in type. So there should be no problem of using `!==`.

Reply:
thank you.
I wanted to ask you if you could add the possibility to create frontend editing

Reply:
What do you mean by frontend editing?

Reply:
With joomla 3 I used https://github.com/vdm-io/Joomla-Component-Builder , this is an example https://aiams.eu/ archive-dei-mulini/segnala-un-mulino.html , it is possible to add in your excellent component builder the possibility to insert data on the frontend side?
This is an example of the use of your program: https: //www.caibo.it/attivita/escursionismo/escursioni-corsi-eventi.html? Ttp = 0

thank you
Ciao
Rino Ruggeri

Reply:
Currently, I've no plan for this. I've planned to create a CLI tool for the extension developers.

Reply:
And if you resolve the reporting issues on your PR then I can merge this PR to the main branch.